test/ProductConstructionService.DependencyFlow.Tests/PullRequestBuilderTests.cs

```diff
@@ -230,6 +247,52 @@ This pull request brings the following source code changes
             """);
     }
 
+    [Test]
+    public void ShouldReturnCorrectDependencyUpdateBlock()
+    {
+        DependencyUpdateSummary newDependency = new()
+        {
+            DependencyName = "Foo.Bar",
+            FromVersion = "",
```

Comment:
I think during runtime, here would be `null`.

Please note that the `DependencyUpdateSummary.cs` file has not nullability enabled so it does not correctly denote possible nulls.